### ISSUE TITLE
Adding license files

### ISSUE THREAD

**Issue by ffontaine** [cifsd-team/ksmbd#552]
It would be great to add license files to the git repository and the official tarball. From my understanding, ksmbd use files under GPL-2.0+ and LGPL-2.1+. Can you confirm that my assumption is correct and add the proper COPYING or LICENSE files (e.g. #144)?

**Comment by namjaejeon** [comment]
Yes, ksmbd is licensed under GPL-2.0+ and LGPL-2.1+. There are SPDX License tags in each *.c/*.h.  Isn't that enough?

**Comment by ffontaine** [comment]
Thanks for your answer.
I would like to add ksmbd to [buildroot](buildroot.org). In buildroot, we declare the license of each package as well as its license files and their hash, for example, for ksmbd-tools:

```
KSMBD_TOOLS_LICENSE = GPL-2.0+
KSMBD_TOOLS_LICENSE_FILES = COPYING

sha256  576540abf5e95029ad4ad90e32071385a5e95b2c30708c706116f3eb87b9a3de  COPYING
```
If there is no license files, we can use the source files but it is less convenient.
